cocotb_test/simulator.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         extra_env=None,
         compile_only=False,
         waves=None,
+        timescale="1ns/1ps",
```

Comment:
```suggestion
        timescale="None",
```

I have feeling this would be better. Do not change default behavior. Apart from this looks OK.